recipes/jsonpickle/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "jsonpickle" %}
+{% set version = "0.9.5" %}
+{% set sha256 = "cc25dc79571d4ad7db59d05ddb7de0d76a8d598cf6136e1dbeaa9361ebcfe749" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - jsonpickle
+
+about:
+  summary: Python library for serializing any arbitrary object graph into JSON. It can take almost any Python object and turn the object into JSON. Additionally, it can reconstitute the object back into Python.
+  home: http://jsonpickle.github.io/
+  license: BSD-3-Clause
+  license_family: BSD
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
+    - robmcmullen
```

Comment:
@robmcmullen -- Are you ok being added as a co-maintainer of the `jsonpickle` package on conda-forge?